builds/snap/snapcraft.yaml

```diff
@@ -0,0 +1,133 @@
+confinement: strict
+grade: stable
+
+apps:
+  easyrpg-player:
+    command: bin/desktop-launch $SNAP/usr/bin/easyrpg-player
+    plugs:
+      - x11
+      - wayland
+      - unity7
+      - opengl
+      - audio-playback
+      - joystick
+      - home
+
+    environment:
+      XDG_DATA_HOME: $SNAP_USER_COMMON/
+      DBUS_FATAL_WARNINGS: 0
+
+layout:
+  /etc/wildmidi:
+    bind: $SNAP/etc/wildmidi
+  /usr/share/midi/freepats:
+    bind: $SNAP/usr/share/midi/freepats
+
+parts:
+  liblcf:
+    plugin: autotools
+    source: https://github.com/EasyRPG/liblcf
+    source-type: git
+    autotools-configure-parameters:
+      - --prefix=/usr
+    override-build: |
+      git fetch
+      snapcraftctl build
+    build-packages:
+      - libexpat1-dev
+      - libicu-dev
+      - icu-devtools
+    stage-packages:
+      - libicu66
+
+  easyrpg-player:   
+    plugin: autotools
+    after: [libwildmidi,liblcf,desktop-glib-only]
+    source: https://github.com/EasyRPG/Player
+    source-type: git
+    autotools-configure-parameters:
+      - --prefix=/usr
+    override-build: |
+      last_committed_tag="$(git tag --list | tac | head -n1)"
+      trimmed_tag="$(echo $last_committed_tag | sed 's/desc\///' | sed 's/git//' | sed 's/^v//')"
+      last_released_tag="$(snap info easyrpg-player | awk '$1 == "latest/beta:" { print $2 }')"
+      # If the latest tag from the upstream project has not been released to
+      # beta, build that tag instead of master.
+      if [ "${trimmed_tag}" != "${last_released_tag}" ]; then
+        git fetch
+        git checkout -f "${last_committed_tag}"
+        snapcraftctl set-version $(git -C ../src tag --list | tac | head -n1 | sed 's/desc\///' | sed 's/git//' | sed 's/^v//')
+      else
+        snapcraftctl set-version $(git describe | sed 's/desc\///')
+      fi
+      snapcraftctl build
+
+    build-packages:
+      - libfmt-dev
+      - libfreetype6-dev
+      - libharfbuzz-dev
+      - libmpg123-dev
+      - libopusfile-dev
+      - libpixman-1-dev
+      - libpng-dev
+      - libsdl2-dev
+      - libsdl2-mixer-dev
+      - libsdl2-ttf-dev
+      - libsndfile1-dev
+      - libspeexdsp-dev
+      - libvorbis-dev
+      - libxmp-dev
+
+    stage-packages:
+      - freepats
+      - libfreetype6
+      - libharfbuzz0b
+      - libmpg123-dev
+      - libstdc++6
+      - libgl1-mesa-dri
+      - libgl1-mesa-glx
+      - libglu1-mesa
+      - libopusfile0
+      - libpixman-1-0
+      - libpng16-16
+      - libsdl2-2.0-0
+      - libsdl2-mixer-2.0-0
+      - libsdl2-ttf-2.0-0
+      - libsndfile1
+      - libspeexdsp1
+      - libvorbis0a
+      - libwayland-egl1-mesa
+      - libwildmidi-config
+      - libxmp4
+      - locales-all
+
+  libwildmidi:
+    plugin: cmake
+    source: https://github.com/Mindwerks/wildmidi
+    source-type: git
+    cmake-parameters:
+      - -DCMAKE_BUILD_TYPE=Release
+      - -DBUILD_SHARED_LIBS=OFF
+      - -DWANT_PLAYER=ON
```

Comment:
- Player component is not needed (also, no alsa library)
- Reason why this library should be static?
- `CMAKE_INSTALL_PREFIX=/usr` should be set to avoid `/usr/local` mess.

Reply:
Fixed.